.github/workflows/main.yml

```diff
@@ -0,0 +1,77 @@
+name: Generate new docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'ember or ember-data'     
+        required: true
+      version:
+        description: 'The version to build, i.e. 3.20.0'     
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Clone the project repo
+        run: |
+          set -euo pipefail
```

Comment:
I think roughly this this is already implied/defaulted for you for action scripts so it shouldn't be necessary

Reply:
Source: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell